src/modules/test/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+add_subdirectory(src)
+
+if (BUILD_TESTS)
+    add_subdirectory(tests)

```

Comment:
In general try to avoid creating too many subdirectories for this. Try to avoid directories whose only purpose are to contain another subdir and have an forwarding empty CMakeList.txt like this.

Please try to simplify the directory structure for this modules/test.

All files can be under this one directory. Then there can be sub-directories for unit-tests for the test, recipes, etc.